src/NUnitFramework/framework/Constraints/EqualityAdapter.cs

```diff
@@ -84,9 +91,9 @@ public override bool AreEqual(object x, object y)
             }
         }
 
-        #endregion
+#endregion
 
-        #region Nested IEqualityComparer Adapter
+#region Nested IEqualityComparer Adapter
```

Comment:
Formatting again, we indent regions. VS 2015 keeps doing this to me too when I add #if statements. I really need to figure out how to disable that :frowning: